src/engraving/libmscore/fret.cpp

```diff
@@ -1522,7 +1522,8 @@ String FretDiagram::screenReaderInfo() const
             }
         }
 
-        //: Omit the "%n " for the singular translation (and the "(s)" too)
+        //: Make sure to translate singular and plural form and whatever else your language has here.
+        //: Best omit the "%n " for the singular translation (and the "(s)" too).
```

Comment:
French uses the singular form for the `n = 0` case (they say the equivalent of "zero dot" rather than "zero dots") so in general it is not safe to tell people to omit the `%n` for any cases, though I appreciate we don't use the `n = 0` case here.

You could say this instead:
> Please provide a translation for the singular and all plural forms available in your language.
> See https://github.com/musescore/MuseScore/wiki/Help-translate-MuseScore#plural-forms

And add an explanation on that page about when it is and isn't safe to use a "wordy" number.

Reply:
0 is not the singular case then, or is it? Whatever: there's no "0" translation in French on Transifex, just 1, many and other. I guess orther does cover the 0 case, if so its  current translation "%n point**s** sur la frette %1" needs to get adjusted.

Still that sentence seems correct, as far as I can tell. Your proposal is too though.

Reply:
We'll have to experiment to see whether MuseScore uses the `1` translation for the `n = 0` case in French.

There may be languages where the `1` case is used for all numbers ending in 1 (`n = 1, 11, 21, 31...`, etc.) or something like that, so I would be wary of encouraging special cases except by experienced translators who know exactly what they are doing.

Reply:
Fine by me, hence I'm using your wording now.

I'd rather leave it to you to adjust https://github.com/musescore/MuseScore/wiki/Help-translate-MuseScore#plural-forms